skimage/filters/lpi_filter.py

```diff
@@ -167,8 +167,16 @@ def forward(data, impulse_response=None, filter_params={},
     return predefined_filter(data)
 
 
+@deprecated(alt_func='skimage.filters.lpi_filter.filter_inverse',
+            removed_version='1.0')
```

Comment:
This `removed_version` doesn't make sense given that we proposed in SKIP 4 that the next version will be 1.0 so it cannot also be removed in 1.0. If we follow SKIP 4 then it would never be removed from `skimage` as there are no more releases after 1.0, so it would get removed in the new `skimage2` package.

Reply:
@grlee77 thanks for pointing that out. According to what we have with the SKIP and the discussion in #6448, I think it should be `removed_version=2.0` here, then.

Reply:
Why don't we just continue as though skimage2 doesn't exist, since we'll likely still make a few releases of skimage1 before that happens?

Reply:
@stefanv works for me, too.

Reply:
> Why don't we just continue as though skimage2 doesn't exist, since we'll likely still make a few releases of skimage1 before that happens?

@stefanv Could you elaborate on what you mean by this? Do we continue making releases on the 1.x branch until all deprecations are done? I'm not saying this is a bad idea but would require amending [SKIP 4](https://github.com/scikit-image/scikit-image/blob/main/doc/source/skips/4-transition-to-v2.rst). Currently, SKIP 4 is only planning for 1.0 and 1.1 where 1.1 is identical save for the migration warning.

Reply:
Plan for now is to go ahead with 0.20 release, possibly 0.21 as well.

Please see [discussion](https://discuss.scientific-python.org/t/a-pragmatic-pathway-towards-skimage2/530/10) on skimage2 transition pathway.